src/sparseml/onnx/utils/helpers.py

```diff
@@ -796,7 +796,12 @@ def get_node_output_nodes(model: ModelProto, node: NodeProto) -> List[NodeProto]
     for output_id in get_node_outputs(model, node):
         nodes.extend(get_nodes_by_input_id(model, output_id))
 
-    return nodes
+    unique_nodes = []
+    for node in nodes:
+        if node not in unique_nodes:
+            unique_nodes.append(node)
+
+    return unique_nodes
```

Comment:
it this fine?
```suggestion
    return list(set(nodes))
```

Reply:
Great suggestion!

Reply:
Had to revert change. The set construction failed w/ onnx nodes